tool/get-license.ts

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env node
+
+// If developing Replicache locally, run `npm link` to link the
+// command. Then you can run this command with `npx get-license`.
+// You should just be able to run `npx get-license` if you are
+// in a package that has Replicache as a local depenency. (Note:
+// if not, the binary lives under node_modules/.bin/, we would
+// need to run it from there instead.)
+
+import * as licensingCLI from '@phritz/licensing/src/cli';
```

Comment:
The export should be `'@phritz/licensing/cli';`  (no `src`)

Reply:
You can achieve this wit h the `exports` field in the package.json for `@phritz/licensing`

Reply:
Per discussion with @aboodman we are publishing the licensing package as typescript sources and depending on its sources as a dev dependency in replicache. We are not depending on its built output in replicache so I think I the I need the `src` here? The compiler complains about not finding it if I omit the src.

Reply:
That sounds right to me.

I think, to make that even simpler we can use a github dependency in package.json

Reply:
I'm going to leave this as an exercise for the future. (I would, however, be interested in a very easy way to have a common package that we could put things like .eslint, prettierrc, asserts.ts, etc in and share. Git submodules?)